react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+
+export default function ToggleButton({ text, color }) {
+  const [isClicked, setIsClicked] = useState(false);
+  console.log('useState:', isClicked);
+
+  function handleClick() {
+    console.log('before setter:', isClicked);
+    setIsClicked(isClicked ? false : true);
+    console.log('after setter:', isClicked);
+  }
+
+  return (
+    <button onClick={handleClick} style={ isClicked ? { backgroundColor: 'white' } : { backgroundColor: color } }>
```

Comment:
Can simplify your ternary a bit.

`style={ backgroundColor: isClicked ? 'white' : color }`

Reply:
Got it!